src/App/Tracker/Controller/Hooks/ReceiveCommentsHook.php

```diff
@@ -318,11 +342,16 @@ protected function updateComment($id)
 		}
 		catch (\Exception $e)
 		{
-			$this->logger->error(
-				'Error loading the database for comment '
-				. $this->hookData->issue->number
-				. ':' . $e->getMessage()
+			$logMessage = sprintf(
+				'Error loading the database for comment %d:%s',
+				$this->hookData->issue->number,
+				$e->getMessage()
 			);
+			$this->logger->error($logMessage);
+			$this->response->error = $logMessage;
+			$this->setStatusCode($e->getCode());
+
+			return false;
```

Comment:
Previously we logged a success message and returned true here - I've changed it to returning false but would appreciate someone reviewing that - unsure if it's related to the event being thrown